qutip/tests/test_subsys_apply.py

```diff
@@ -147,11 +114,11 @@ def test_ComplexSuperApply(self):
         analytic_result[1] = Qobj(
             vector_to_operator(
                 superop @ operator_to_vector(analytic_result[1])
-            ).full())
+            ))#.full())
         analytic_result[3] = Qobj(
             vector_to_operator(
                 superop @ operator_to_vector(analytic_result[3])
-            ).full())
+            ))#.full())
```

Comment:
Commented out `.full()` should be removed before merge.

Reply:
Removed